dependencies.gradle

```diff
@@ -23,6 +23,7 @@ ext.deps = [
     javaPoet                 : 'com.squareup:javapoet:1.9.0',
     jsr305                   : 'com.google.code.findbugs:jsr305:3.0.2',
     okio                     : 'com.squareup.okio:okio:1.13.0',
+    gradleMavenPublishPlugin : 'com.vanniktech:gradle-maven-publish-plugin:0.8.0',
```

Comment:
@adellhk unless you use the 0.9.0 snapshot it won't properly include kotlin sources. This is documented in the mvrx setup that I linked you too.